src/init/AppRunner.ts

```diff
@@ -33,41 +35,65 @@ export class AppRunner {
   /**
    * Starts the server with a given config.
    * This method can be used to start the server from within another JavaScript application.
+   *
    * Keys of the `variableBindings` object should be Components.js variables.
    * E.g.: `{ 'urn:solid-server:default:variable:rootFilePath': '.data' }`.
    *
+   * `settings` corresponds to values that could be passed to the CLI.
+   * E.g.: `{ rootFilePath: '.data' }`.
+   * Shorthand parameter names can not be used, so `{ f: '.data' }` would not work.
+   *
+   * If a certain would set a specific variable,
+   * and both are present in `variableBindings` and `settings`,
+   * the value in `variableBindings` will take priority.
```

Comment:
If `variableBindings` take priority, the answer to my previous question should probably be "keys with default value", because I expect CLI to override variables.

Reply:
Those 2 functions (`run` and `start`) are used to start the server through code instead of the CLI. The `settings` object fulfils the same role as the CLI parameters here. It's just that additionally you can also set the components.js variable bindings directly instead of relying on those parameters being parsed in a certain way.

Reply:
> The settings object fulfils the same role as the CLI parameters here.

Maybe consistently `cliArguments` or similar here then? Even if from code?
Alternatively, `shorthandParams` or `shortParams` or so.

Reply:
Might indeed be clearer if we refer to them as CLI values even if they aren't really from the CLI. The name `Settings` was originally chosen to generalize this kind of object, but in practice that is what it is. Would you also change the type name then?

Related discusison: https://github.com/CommunitySolidServer/CommunitySolidServer/pull/1100#discussion_r791800078

Reply:
Yes, I think that would improve clarity indeed.